server/services/reportService.ts

```diff
@@ -63,6 +73,20 @@ export default class ReportService {
     return report
   }
 
+  async getReportInProgress(bookingId: number): Promise<inProgressReport | null> {
```

Comment:
This get the current 'in progress' report

Reply:
Think you can get rid of this and use:`DraftReportService.getCurrentDraft`

Which does what you describe

Reply:
replaced getReportInProgress with DraftReportService.getCurrentDraft